genesis/options/solvers.py

```diff
@@ -400,7 +401,7 @@ class RigidOptions(Options):
     enable_mujoco_compatibility: bool = False
 
     # GJK collision detection
-    use_gjk_collision: bool = True
```

Comment:
can we still use gjk by default?

Reply:
GJK is much slower than MPR+SDF without any proven benefit at the moment. It is clearly a more future-proof algorithm than MPR+SDF but I think it requires more polishing before being pushed to the user by default. Still we do not have any recorded failure cases for the latest version of MPR+SDF in practice. In my opinion, its dramatic impact on runtime speed makes a worst default choice than MPR+SDF for most users that do not need extreme robustness.